backend/src/test/java/com/bakdata/conquery/util/NonPersistentStore.java

```diff
@@ -7,7 +7,9 @@
 
 import com.bakdata.conquery.io.storage.Store;
 import com.bakdata.conquery.io.storage.xodus.stores.SerializingStore;
+import lombok.ToString;
 
+@ToString
 public class NonPersistentStore<KEY, VALUE> implements Store<KEY, VALUE> {
 
     private final HashMap<KEY,VALUE> map = new HashMap<>();

```

Comment:
pew ist das nicht ganz schön verbose in den Tests?

Reply:
> pew ist das nicht ganz schön verbose in den Tests?

gerade in den Tests doch eigentlich nicht, da sind ja idR nur eine Hand voll Einträge drin, das hat mir beim debuggen gut geholfen.